src/components/action-popover/action-popover.pw.tsx

```diff
@@ -475,7 +475,7 @@ test.describe("check functionality for ActionPopover component", () => {
     await actionPopoverButton(page).click();
 
     // check download item has 'download' property
-    const downloadItem = page.getByRole("menuitem", { name: "Download" });
+    const downloadItem = page.getByRole("link", { name: "Download" });
```

Comment:
I can also see Playwright is passing so, I must be missing something

Reply:
You are, but no worries, it's really easy to miss - iirc I first made the role `"button"` but then the test failed. The reason is the Download "item" is actually a link - if you look at the story, the `ActionPopoverItem` has an `href`, which makes  `ActionPopoverItem` render an anchor instead of a button.